index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio: Mobile First</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header class="container-header flex-row">
+    <div class="logo"><img src="image/my-logo.png" alt="My Logo"></div>
```

Comment:
Kindly wrap your logo in an `<a>` tag🙏. This will improve the semantics of your page, as the logo typically links to the homepage